src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -35,9 +37,10 @@
 public class ModelManager extends ComponentManager implements Model {
     private static final Logger logger = LogsCenter.getLogger(ModelManager.class);
     private static final Predicate<ReadOnlyParcel> deliveredPredicate = p -> p.getStatus().equals(Status.COMPLETED);
+    private static final Index TAB_ALL_PARCELS = Index.fromZeroBased(0);
```

Comment:
![image](https://user-images.githubusercontent.com/17818981/32423140-a5fa02a4-c2df-11e7-962e-640ac44ce384.png)

Please use the variables i created here to get the index lol

Reply:
the constants*